packages/manager/modules/bm-server-components/src/general-information/inputs/template.html

```diff
@@ -90,23 +90,15 @@ <h5 data-translate="server_configuration_installation_inputs_title"></h5>
         >
         </textarea>
         <!-- sshPubKey -->
-        <div
-            data-ng-switch-when="sshPubKey"
-            class="float-right sshkey-switch"
-            data-sshkey-switch
-            data-sshkey-switch-key="{{$ctrl.installation.input[input.name]}}"
-        ></div>
-        <textarea
-            data-ng-switch-when="sshPubKey"
-            class="oui-input form-control"
-            data-ng-model="$ctrl.installation.input[input.name]"
-            data-ng-required="input.mandatory"
-            rows="{{$ctrl.inputRules.sshPubKey.maxsize}}"
-            name="{{input.name}}"
-            data-ng-pattern="$ctrl.inputRules.sshPubKey.pattern"
-            placeholder="{{:: 'server_configuration_installation_inputs_item_placeholder' | translate:{ t0: 'sshPubKey' } }}. {{:: 'server_configuration_installation_inputs_item_placeholder_example' | translate:{ t0: $ctrl.inputRules.sshPubKey.placeholder } }}"
-        >
-        </textarea>
+        <div data-ng-switch-when="sshPubKey">
```

Comment:
![image](https://github.com/user-attachments/assets/ae6f10b6-ea3f-4823-bd77-4a413c0404f7)
Add a vertical margin between the component and the question label from current template